include/RAJA/policy/cuda/reduce.hpp

```diff
@@ -641,15 +639,13 @@ RAJA_DEVICE RAJA_INLINE bool grid_reduce(RAJA::expt::detail::Reducer<OP, T>& red
       temp = OP{}(temp, red.device_mem.get(i));
     }
 
-    temp = block_reduce<OP>(temp, OP::identity());
+    temp = block_reduce<ThreadIterationGetter, OP>(temp, OP::identity());
 
     // one thread returns value
     if (threadId == 0) {
       *(red.devicetarget) = temp;
     }
   }
-
-  return lastBlock && threadId == 0;
```

Comment:
Shouldn't we keep this? We should still want to know whether we've hit the last block?

Reply:
That is a holdover from the non-expt version where it is used, but here it is not used. I'll clean this up in a subsequent PR where the two versions will be recombined.